RIOT-OS/RIOT
# Fix kconfig models breaking nightlies

### Contribution description

It seemed that the `tinyusb` package exposes some issues with the kconfig modelling for some `stm32`s as well as improper modelling in the `tinyusb` package.

This broke the extensive kconfig tests in nightlies.  


### Testing procedure

The remove me commit forces the nightly test, it is limited to only the boards failing, maybe we want to check it with a full nightly run, but as this is a fix I would be ok removing the REMOVEME and do a regular test.

### Issues/PRs references
